R/indicator_native_range_year.R

```diff
@@ -0,0 +1,123 @@
+#' Create interactive plot for counts per native region and year of introduction
```

Comment:
I would not use the word `counts` in the title as it is quite vague. I suppose you speak about alien species?
I would replace `counts` with `number of alien species`.

Reply:
done